index.html

```diff
@@ -1345,43 +1345,14 @@ <h2 property="schema:name">Storage</h2>
             <div datatype="rdf:HTML" property="schema:description">
               <p>
                 The <code>pim:storage</code> predicate is used to indicate where
-                a WebID owner stores their data. An app wanting to access the
-                WebID owner's Pod should find its location using the
-                <code>pim:storage</code> predicate. 
+                a WebID owner stores their data. When an application needs to access the
+                WebID owner's storage, it can find its location using the
+                <code>pim:storage</code> predicate.
```

Comment:
```suggestion
               available to the agent to use.
```

I think the second sentence is superfluous.